spec/assoc.test.sh

```diff
@@ -33,7 +33,7 @@ declare -A assoc=([k1]=foo [k2]='spam eggs')
 for v in "${assoc[@]}"; do echo $v; done | sort
 for v in "${!assoc[@]}"; do echo $v; done | sort
 
-# disallow this in OSH?  Changing type?
+# Bash 5.1 assoc=(key value). Bash 4.4 does not implement this.
 
 assoc=(foo 'spam eggs')
 argv.py "${assoc[@]}"
```

Comment:
It might be clearer to test the output of `declare -p assoc` after this?

Reply:
OK, I fixed it in 6c2014b4d.